cep-XXXX.md

```diff
@@ -0,0 +1,201 @@
+or the channel. It MUST match the following syntax:
+
+```text
+<package name>-<version string>-<build string>
+```
+
+Distribution strings apply to both distributable and virtual packages. They are used as the name of the
+directories where artifacts are extracted in the package cache, for example.
+
+> Note: Despite the similarity, distribution strings are not `MatchSpec`-like specifiers and
+> MUST NOT be used as such.
+
+### Identifying channels
+
+A conda channel is defined as a URL where one can find one or more `repodata.json` files arranged in one
+subdirectory (_subdir_) each. `noarch/repodata.json` MUST be present to consider the parent location a channel.
+
+#### Channel base URLs and names
+
+The base URL for the arbitrary location of a repodata file is defined as:
+
+```text
+<scheme>://[<authority>][/<path>/][/label/<label name>]/<subdir>/repodata.json
+```
+
+with `<scheme>`, `<authority>` and `<path>` defined by [RFC
+3986](https://datatracker.ietf.org/doc/html/rfc3986#section-3.2).
+
+Taken the channel definition above, the base URL without trailing slashes is thus:
+
+```text
+<scheme>://[<authority>][/<path>/][/label/<label name>]
+```
+
+For example, given `https://conda.anaconda.org/conda-forge/noarch/repodata.json`, the part leading
+to `noarch/repodata.json` and thus base URL is `https://conda.anaconda.org/conda-forge/conda-forge`. For
+local repodata such as `file:///home/username/channel/noarch/repodata.json`, the
+channel base URL is `file:///home/username/channel`.
+
+For convenience, the channel _name_ is defined as the concatenation of `scheme`, `authority` and
+`path` components. The `<scheme>://` part MAY be omitted if `scheme` is one of `http`, `https` or
+`file`. At least one of `authority` or `path` SHOULD be present. In their absence, the channel name
+MUST be considered empty, regardless the scheme. Empty channel names SHOULD NOT be used.
+
+When present, each path component MUST only contain lowercase ASCII letters, numbers, underscores,
+periods, and dashes. They MUST NOT start with a period or a dash. They SHOULD start and end with a
+letter or a number. If present, each path component MUST match this regex:
+
+```re
+^[a-z0-9_][a-z0-9_.-]*$
+```
+
+The maximum length of an individual path component in a channel base URL MUST NOT exceed 128 characters.
+The maximum length of a channel base URL SHOULD NOT exceed 256 characters.
+
+#### Subdir names
+
+Channel subdir names MUST either be the literal `noarch` or a string following the syntax `{os}-{arch}`,
+where `{os}` and `{arch}` MUST only consist of lowercase ASCII letters and numbers. Non-`noarch` subdirs
+MUST match this regex: `^[a-z0-9]+-[a-z0-9]+$`.
+
+The maximum length of a subdir name MUST NOT exceed 32 characters.
+
+#### Label names
+
+Channel label names MUST only consist of ASCII letters, digits, underscores, hyphens, forward slashes,
+periods, colons and whitespace. They MUST start with a letter. They MUST match this regex:
```

Comment:
I'm inclined to disallow colons, since they're disallowed on CIFS and Windows (except for drive letters) and get interpreted as directory delimiters on macOS.

Reply:
Applied suggestions.